src/components/Capture/capture.js

```diff
@@ -92,12 +92,9 @@ class Capture extends Component {
     })
   }
 
-  createPayload = (blob, base64) => ({
-    id: randomId(), blob, base64
-  })
+  createPayload = (blob, base64) => ({blob, base64})
 
-  createJSONPayload = ({id, base64}) =>
-    JSON.stringify({id, image: base64})
+  createJSONPayload = ({base64}) => JSON.stringify({image: base64})
```

Comment:
These `createPayload` functions aren't really doing anything. I would get rid of them